data/data/google/main.tf

```diff
@@ -0,0 +1,18 @@
+locals {
```

Comment:
the directory needs to be `data/data/gcp` right?

Reply:
Does it? Where the aws provider is "aws", the gcp provider is "google".

Reply:
> Does it? Where the aws provider is "aws", the gcp provider is "google".

the platform in terms of installer is `gcp`, the terraform plugin name is `google`

https://github.com/openshift/installer/blob/63c63a0015f8f7cd894a93c0a6c0d6565ee8d05a/pkg/asset/cluster/cluster.go#L73
and
https://github.com/openshift/installer/blob/63c63a0015f8f7cd894a93c0a6c0d6565ee8d05a/pkg/terraform/terraform.go#L93
makes installer copy (unpack) the `data/data/<platform>` to tmp directoty where the terraform is run

Reply:
@abhinavdahiya Done, PTAL